includes/discovery/os/metronid.inc.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+/*
+ * LibreNMS Acceidan MetroNID OS Discovery module
+ *
+ * Copyright (c) 2016 Chris A. Evans <thecityofguanyu@outlook.com>
+ *
+ * This program is free software: you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License as published by the
+ * Free Software Foundation, either version 3 of the License, or (at your
+ * option) any later version.  Please see LICENSE.txt at the top level of
+ * the source code distribution for details.
+ */
+
+
+if (!$os) {
+    if (preg_match('/AMN-/', $sysDescr)) {
+        $os = 'accedian';
```

Comment:
Is this the right name for the OS? Accedian is the company name and detection should be named after what the OS is.